XBVC/emitters/templates/c_interface.jinja2

```diff
@@ -62,7 +111,7 @@ static int xbvc_encode_bit_vector(void *src, uint8_t *dst, uint8_t rt)
 
 static int xbvc_decode_bit_vector(uint8_t *src, void *dst, uint8_t rt)
 {
-    int mul = 1;
+    uint64_t mul = 1;
```

Comment:
I think I'm missing the significance of this change. Could you explain briefly?

Reply:
Sure. I've experienced overflow issues in the past when encoding large numbers.  I just wanted to use the largest container possible here since it has low performance impact, but saves us from overflow errors in the field

Reply:
Thanks! Makes perfect sense.